src/modules/collection/selectors.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { ChainId, WearableCategory } from '@dcl/schemas'
+import { getChainIdByNetwork } from 'decentraland-dapps/dist/lib/eth'
+import { RootState } from 'modules/common/types'
+import { SyncStatus } from 'modules/item/types'
+import { getStatusByCollectionId } from './selectors'
+
+jest.mock('decentraland-dapps/dist/lib/eth')
+const mockGetChainIdByNetwork = getChainIdByNetwork as jest.Mock
+
+describe('when getting status by item id', () => {
+  it('should return the status for each published item', () => {
+    mockGetChainIdByNetwork.mockReturnValue(ChainId.MATIC_MAINNET)
```

Comment:
Lets use `.mockReturnValueOnce` just in case.

Reply:
In this particular test is unnecessary, but if you were to add a new one with another `mockReturnValue`, the second would not work.

Reply:
The test fails without it because it calls that mock 3 times